### DISCUSSION TITLE
iPhone拍照實用小技巧

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#768] [General]
- 音量鍵當快門
這點在橫向拍照的時候很好用，就不會有少一隻手的困擾了
- 長按快門就是錄影模式
拍小短片的時候很實用 鬆開就結束跟instagram的限時錄法很像
- 按下快門迅速往左滑就可連拍
這個小技巧很實用而且也可看到張數
- 光學變焦放大、縮小，畫質不會下降
我覺得這點是最好用的 上課常常要拍黑板 一隻手就能放大 畫質也不錯
- 關原況模式
原況照片大概是許多人手機容量不夠的主因
- 手動對焦
輕按螢幕就能對焦在想要的地方了 同時還能調亮度
[iPhone拍照實用小技巧](https://news.tvbs.com.tw/life/1929562)

**Comment by a45320614** [comment]
可惜安卓的手機用不到如此實用的iPhone小技巧

**Comment by williamyen0623** [comment]
哦耶！我也可以當網美了！好開心！

**Comment by manipulateed** [comment]
雖然是簡單的小技巧，卻能幫助直男的生存，感謝分享。

**Comment by evanchou1204** [comment]
需要的可能不是拍照小技巧，是另一篇文裡面的AI修圖。

**Comment by danwan1027** [comment]
感謝分享實用的拍照小技巧！

**Comment by Pietra1226** [comment]
感謝分享這個小技巧，以後終於不會被罵了^_^

**Comment by bonny123865** [comment]
長按快門變錄影真的很方便，然後我用到現在還真的不知道原況的照片用途是什麼

**Comment by AnAnistired** [comment]
雖然大部分小技巧我都知道，但用到的時機真的少

**Comment by wenchi9023** [comment]
看完好像重新換手機了，好多之前沒注意到的小技巧

**Comment by ru0372572** [comment]
每次按音量鍵拍出來都是糊的QQ

**Comment by Chiway1229** [comment]
我用了音量鍵照相，結果超模糊
難道是我手抖太嚴重嗎...

**Comment by ghost** [comment]
很實用的小技巧👍，我等我買了iphone會好好運用的，嗯...等我買了之後

**Comment by yuxinlin93** [comment]
對焦用的那個小方框旁邊有一個可以調動的軸，是拿來調整曝光度的，明明就在那裏但很多人都會忽略它，實用他會拍出很多很酷的照片!

**Comment by Kotori0501** [comment]
感謝分享
但我沒有用過iPhone
希望工作後有錢可以以買隻是試試看

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
希望安著也能做出些像蘋果的拍照小技巧

**Comment by Zoe06060606** [comment]
原況照片真的佔了我相簿空間不少的容量 以後拍照一定要注意

**Comment by Evan0979** [comment]
學會了,請問iphone要去哪裡領QQ

**Comment by StevenWanglolz** [comment]
@evanchou1204 身為亞服榜上排名872的專業攝影師還有snapchat大師，這些技巧我早就精通了，盼望你有朝一日也能跟上的步伐!

**Comment by CPCpyt** [comment]
學會了，感謝分享！趕緊出門拍照試試！拍出好作品！
